Description: Fix type in code editor
 Fix a typo in the source code which was reported by lintian.
 .
 godot (2.1.4-stable-1) UNRELEASED; urgency=medium
 .
   * Initial release (Closes: #793057)
   * This is my first package
Author: Kienan Stewart <kienan.stewart@burntworld.ca>
Bug-Debian: https://bugs.debian.org/793057

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2017-12-28

--- godot-2.1.4-stable.orig/editor/code_editor.cpp
+++ godot-2.1.4-stable/editor/code_editor.cpp
@@ -241,7 +241,7 @@ void FindReplaceBar::_replace_all() {
 	}
 
 	text_edit->set_v_scroll(vsval);
-	set_error(vformat(TTR("Replaced %d Ocurrence(s)."), rc));
+	set_error(vformat(TTR("Replaced %d Occurrence(s)."), rc));
 }
 
 void FindReplaceBar::_get_search_from(int &r_line, int &r_col) {
